src/merkle/store/mod.rs

```diff
@@ -85,11 +85,12 @@ impl MerkleStore {
     /// Creates an empty `MerkleStore` instance.
     pub fn new() -> MerkleStore {
         // pre-populate the store with the empty hashes
-        let subtrees = EmptySubtreeRoots::empty_hashes(64);
+        let subtrees = EmptySubtreeRoots::empty_hashes(255);
```

Comment:
We could alternatively create a new method in `EmptySubtreeRoots` that will return all the empty roots

Reply:
I don't understand, this method returns the empty roots, no?

Reply:
ah! you mean a method that receives no arguments?